documentation/asciidoc/accessories/iqaudio/getting_started.adoc

```diff
@@ -2,24 +2,24 @@
 
 === Hardware and wiring
 
-For this project, any small passive speaker should be sufficient, we're using one available https://shop.pimoroni.com/products/3-speaker-4-3w?variant=380549926[here], which is a 3" 4Ω - 5W Speaker. We have also used an illuminated momentary push button button and a laser cut box to house all the components, but both are entirely optional. This example will work just using the Codec Zero on-board button, which is pre-wired to GPIO 27. Alternatively, you may use any momentary push button, such as those available https://shop.pimoroni.com/products/mini-arcade-buttons?variant=40377171274[here].
+For this project, any small passive speaker should be sufficient. We're using one available https://shop.pimoroni.com/products/3-speaker-4-3w?variant=380549926[here], which provides 5W of power at 4Ω. We have also used an illuminated momentary push button, and a laser-cut box to house all the components; but both are entirely optional. This example will work just using the Codec Zero's on-board button, which is pre-wired to GPIO 27. (Alternatively, you can use any momentary push button, such as those available https://shop.pimoroni.com/products/mini-arcade-buttons?variant=40377171274[here].)
 
 image::images/wiring.jpg[width="80%"]
 Note - BRIAN TO ADD LABELS TO THIS PHOTO - GPIO 27, +5V, and Ground x2.
 
-Use a small flat head screwdriver to attach your speaker to the screw terminals. For the additional push button, solder the button wires directly to the Codec Zero pads as indicated on GPIO pin 27 and Ground for the switch, and +5V and Ground for the LED, if necessary. 
+Use a small flat-head screwdriver to attach your speaker to the screw terminals. For the additional push button, solder the button wires directly to the Codec Zero pads as indicated, using GPIO pin 27 and Ground for the switch, and +5V and Ground for the LED, if necessary. 
 
 === Setting up your Raspberry Pi
 
-In this example, we are using Raspberry Pi OS Lite. Our guides on https://www.raspberrypi.com/documentation/computers/getting-started.html#installing-the-operating-system[Getting Started] cover this topic in great detail. Make sure that you update your operating system before proceeding and follow the instructions provided for Codec Zero Configuration, including the commands to enable the on-board microphone and speaker output.
+In this example, we are using Raspberry Pi OS Lite. Our guides on https://www.raspberrypi.com/documentation/computers/getting-started.html#installing-the-operating-system[Getting started] cover this topic in great detail. Make sure that you update your operating system before proceeding and follow the instructions provided for Codec Zero configuration, including the commands to enable the on-board microphone and speaker output.
```

Comment:
Ideally this should be a relative URL rather than an absolute URL.

Reply:
Yup. We should use an `xref:` here not an absolute link.

Reply:
If you're going to lower case the "s" you need to also lower case the "g"?

Reply:
No, because it's title case. (I do love arguing about capitalisation with you though, because I usually win, so keep it up.)

Reply:
Honestly think it should be re-worded to, 

```
In this example, we are using Raspberry Pi OS Lite. Our https://www.raspberrypi.com/documentation/computers/getting-started.html#installing-the-operating-system[getting started] guides cover this topic in great detail.
```

There's no real reason to Yoda it in the way it's written right now?

Reply:
The text should be readable and make sense grammatically if the link is taken away. Right now that it's the case. The title case thing relies on the fact that this is a link to make it make sense.

Reply:
Why don't we leave this until we're in the same room on Wednesday so this takes two minutes rather than a distributed hour of typing?